eventhub/Module.md

```diff
@@ -1,55 +1,22 @@
-# Ballerina Azure Event Hubs Module
+## Overview
 
-Connects to Microsoft Azure Event Hubs using Ballerina.
+Azure Event Hub Ballerina Connector is used to connect with the Azure Event Hubs via Ballerina language easily 
+to ingest millions of events per second so that you can process and analyze the massive amounts of data produced 
+by your connected devices and applications. Once data is collected into an Event Hub, it can be transformed and stored 
+using any real-time analytics provider or batching/storage adapters.
 
-# Module Overview
-
-Azure Event Hubs Ballerina Connector is used to connect with the Azure Event Hubs to ingest millions of events per 
-second so that you can process and analyze the massive amounts of data produced by your connected devices and 
-applications. Once data is collected into an Event Hub, it can be transformed and stored using any real-time 
-analytics provider or batching/storage adapters.
-
-Azure Event Hub Ballerina connector supports Event hub service operations like sending an event, sending batch events, 
+This module supports Event hub service operations like sending an event, sending batch events, 
```

Comment:
like -> such as

Reply:
This module supports the entire Azure Event Hub Service REST API right?

Reply:
The last statement is contradictory. Does it support only sending events? or supports other operations as well?

Reply:
Yes it supports the entire Azure Event Hub Service REST API. I removed the contradictory sentence in df7c75b . Thank you.